[2024-08-11T20:59:45.946+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-11T20:59:45.956+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_table_drop.weather_data_table_drop manual__2024-08-11T20:59:44.652670+00:00 [queued]>
[2024-08-11T20:59:45.959+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_table_drop.weather_data_table_drop manual__2024-08-11T20:59:44.652670+00:00 [queued]>
[2024-08-11T20:59:45.959+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-11T20:59:45.964+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): weather_data_table_drop> on 2024-08-11 20:59:44.652670+00:00
[2024-08-11T20:59:45.970+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=252) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-11T20:59:45.971+0000] {standard_task_runner.py:64} INFO - Started process 254 to run task
[2024-08-11T20:59:45.971+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_table_drop', 'weather_data_table_drop', 'manual__2024-08-11T20:59:44.652670+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/delete_openmeteo_dag.py', '--cfg-path', '/tmp/tmpmxkwd98p']
[2024-08-11T20:59:45.973+0000] {standard_task_runner.py:91} INFO - Job 104: Subtask weather_data_table_drop
[2024-08-11T20:59:46.005+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_table_drop.weather_data_table_drop manual__2024-08-11T20:59:44.652670+00:00 [running]> on host 14d759344545
[2024-08-11T20:59:46.046+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_table_drop' AIRFLOW_CTX_TASK_ID='weather_data_table_drop' AIRFLOW_CTX_EXECUTION_DATE='2024-08-11T20:59:44.652670+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-11T20:59:44.652670+00:00'
[2024-08-11T20:59:46.048+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-11T20:59:46.048+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-11T20:59:46.048+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/delete_table.py --connection postgresql+psycopg2://***:***@postgres/***']
[2024-08-11T20:59:46.053+0000] {subprocess.py:86} INFO - Output:
[2024-08-11T20:59:46.495+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-11T20:59:46.496+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-11T20:59:46.508+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_table_drop, task_id=weather_data_table_drop, run_id=manual__2024-08-11T20:59:44.652670+00:00, execution_date=20240811T205944, start_date=20240811T205945, end_date=20240811T205946
[2024-08-11T20:59:46.528+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-11T20:59:46.537+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-11T20:59:46.538+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
